packages/migrations/src/Command/GenerateMigrationCommand.php

```diff
@@ -122,48 +122,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
      * @param \Symfony\Component\Console\Style\SymfonyStyle $io
      * @return \Shopsys\MigrationBundle\Component\Doctrine\Migrations\MigrationsLocation
      */
-    private function chooseMigrationLocation(SymfonyStyle $io)
+    private function chooseMigrationLocation(SymfonyStyle $io): MigrationsLocation
```

Comment:
maybe use of `realpath` would be nice on the line 105 and 114 (`getMigrationFilePath()`). 
Turning 
```
Migration file "/Volumes/work/shopsys/framework/packages/framework/src/DependencyInjection/../Migrations/Version20220321155459.php" was saved (568 B).
```
to
```
Migration file "/Volumes/work/shopsys/framework/packages/framework/src/Migrations/Version20220321155604.php" was saved (568 B).
```

Reply:
changed in a separate commit - f16982209